### PR TITLE
[DOCFIX] Improve HBase doc

### PR DESCRIPTION
**Pull request by jiacheliu3**
This doc change updates how Alluxio can work with HBase 2.0. One extra parameter needs to be configured. Minor format updates besides that.

### CODE REVIEW

File: docs/en/compute/HBase.md [hunk 1]
```diff
@@ -30,29 +30,42 @@ Therefore, the configuration of Alluxio is done mostly in HBase configuration fi
...
+
+```xml
+<property>
+  <name>hbase.unsafe.stream.capability.enforce</name>
+  <value>false</value>
+</property>
+```
+
+> This will disable HBase new stream capabilities (hflush/hsync) used for WAL.
+HBASE-14790, HBASE-15536
```
Comment (jiacheliu3) [inline docs/en/compute/HBase.md]: This has to be configured for HBase >2.0. The related changes are HBASE-14790 and HBASE-15536. Should I leave a ref to those changes?

File: docs/en/compute/HBase.md [hunk 2]
```diff
@@ -30,29 +30,58 @@ Therefore, the configuration of Alluxio is done mostly in HBase configuration fi
...
+</property>
+<property>
+  <name>fs.alluxio-ft.impl</name>
+  <value>alluxio.hadoop.FaultTolerantFileSystem</value>
+</property>
+<property>
+  <name>fs.AbstractFileSystem.alluxio.impl</name>
+  <value>alluxio.hadoop.AlluxioFileSystem</value>
+</property>
+```
```
Comment (jiacheliu3) [inline docs/en/compute/HBase.md]: FS implementation classes added here. I'm not sure if this is compulsory but all docs indicate that they are needed.

File: docs/en/compute/HBase.md [hunk 3]
```diff
@@ -30,29 +30,58 @@ Therefore, the configuration of Alluxio is done mostly in HBase configuration fi
...
+You also need to add the FS implementation classes to HBase configuration. These classes are provided in Alluxio Client jar.
+
+```xml
+<property>
+  <name>fs.alluxio.impl</name>
+  <value>alluxio.hadoop.FileSystem</value>
+</property>
+<property>
+  <name>fs.alluxio-ft.impl</name>
+  <value>alluxio.hadoop.FaultTolerantFileSystem</value>
```
Comment (bf8086) [inline docs/en/compute/HBase.md]: Does this class still exist?

Comment (jiacheliu3) [reply docs/en/compute/HBase.md]: This is an excellent question.

This class has been marked Deprecated as of 1.6 and removed in https://github.com/Alluxio/alluxio/commit/3d7d18dbe280c6719c779d7d55f2da4f746b62a4. I'll remove it accordingly.

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Comment by apc999** [comment]
alluxio-bot, cherry-pick this to branch-2.1 please

**Pull request merged**